### DISCUSSION TITLE
Automatic failover

### DISCUSSION THREAD

**Discussion by xifel** [aembke/fred.rs#175] [General]
Can fred be configured to failover automatically if the master in a sentinel setup fails?

**Comment by aembke** [comment]
Hi @xifel, it should work that way out of the box if you use a sentinel server config and a reconnect policy.

Let me know if you have any issues though and I'll take a look.

**Reply by xifel** [reply]
Hi @aembke, thank you for your answer. I was missing the reconnect policy. After adding it to the builder it works like a charm.
Thank you for the great crate!
